integration-tests/common/gauntlet_plus_plus_common.go

```diff
@@ -55,3 +109,69 @@ func (m *OCRv2TestState) setConfigDetailsWithGpp(ocrAddress string) error {
 	_, err = m.Clients.GauntletPPClient.SetConfigDetails(string(parsedConfig), ocrAddress)
 	return err
 }
+
+func (m *OCRv2TestState) setConfigDetails(ocrAddress string) error {
```

Comment:
can you add a todo comment here explaining that it uses legacy gauntlet but will use G++ in the future